tests/test-tokenizer.py

```diff
@@ -331,3 +331,14 @@ def test_tokenize(self):
                           t(tt.WORD, 'a', [0, 1]),
                           t(tt.WORD, "'b  '", [2, 7], set([flags.word.QUOTED])),
                           t(tt.WORD, 'c', [8, 9])])
+
+    def test_variables(self):
```

Comment:
please add a test for the invalid case where you a variable name starts with a digit

Reply:
How to write an invalid case?

Reply:
try to parse something like `0var=1`

Reply:
Can you elaborate more? Parse in the test case?

Reply:
Yes, please add a test case for the problematic assignment which shows that parsing fails. If you're having trouble, I can do it.

Reply:
I added a failed case, but I am not sure if it is a correct way to do it.